examples/graphql-context/graphql_context.bal

```diff
@@ -30,26 +29,23 @@ service /graphql on new graphql:Listener(4000) {
     function init() {
         // Initialize the `person` value.
         self.person = new("Walter White", 51, 737000.00);
-
     }
 
     // Resource functions can be defined without a context parameter.
     resource function get greet() returns string {
         return "Hello, world";
     }
 
-    // If the context is needed, it should be defined as the first paramter of
-    // the resolver function.
+    // If the context is needed, it should be defined as the first parameter of the resolver
+    // function.
```

Comment:
Line breaks?

Reply:
Yes, we need not add line breaks up to the character limit of 100.